int8 FRAME_ODOM=1
int8 FRAME_MAP=2
int8 FRAME_BASE=3

geometry_msgs/PoseStamped[] plan
int8 frame
---
int8 STATUS_ACCEPTED=1
int8 STATUS_REJECTED=0
int8 status